ethereum/statetransition/src/main/java/tech/pegasys/teku/statetransition/validation/DataColumnSidecarGossipValidator.java

```diff
@@ -102,6 +110,16 @@ private DataColumnSidecarGossipValidator(
     this.miscHelpersEip7594 = miscHelpersEip7594;
     this.kzg = kzg;
     this.receivedValidDataColumnSidecarInfoSet = receivedValidDataColumnSidecarInfoSet;
+    this.totalDataColumnSidecarsProcessingRequestsCounter =
+            metricsSystem.createCounter(
+                    TekuMetricCategory.BEACON,
+                    "beacon_data_column_sidecar_processing_requests_total",
+                    "Total number of data column sidecars submitted for processing");
+    this.totalDataColumnSidecarsProcessingSuccessesCounter =
+            metricsSystem.createCounter(
+                    TekuMetricCategory.BEACON,
+                    "beacon_data_column_sidecar_processing_successes_total",
```

Comment:
`beacon` is not needed here and there, it will be added automatically by the tag, so now it's beacon_beacon_data...